Python/iSenseAPI.py

```diff
@@ -15,13 +17,19 @@
 # Gets info about a project, if recur is true it returns extra info
 def getProject(projID, recur=False):
     url = BASE_URL + 'projects/' + str(projID) +'?recur=' + str(recur).lower()
-    return requests.get(url)
+    request = requests.get(url)
+
+    dictionary = json.loads(request.text)
+    return dictionary
```

Comment:
The dictionary provides a key called "status" that would give them this information. I want to keep the api as low level as possible so that it won't ever have to be modified to catch new errors.
The wrapper should be the one checking the status of each api call and then throwing appropriate exceptions to be caught by the user.